rotkehlchen/rotkehlchen.py

```diff
@@ -435,6 +441,15 @@ def deactivate_premium_status(self) -> None:
         self.accountant.deactivate_premium_status()
         self.chain_manager.deactivate_premium_status()
 
+    def activate_premium_status(self, premium: Premium) -> None:
+        """Activate the premium in the current session if was deactivated"""
+        if self.premium is not None:
```

Comment:
Why this check here?

Reply:
I believe is not possible so will just remove it